packages/cli/src/commands/dev/consts.ts

```diff
@@ -0,0 +1 @@
+export const TUNNEL_URL = 'https://novu.sh/api/tunnels';
```

Comment:
🤓 It's not worth the extra import and file for just this. Consider moving it to the dev.ts file.

Reply:
yep agree i was extracting one by one, initially, i was not aware how many would go for each file 😅